extensions/webdav-uploader/src/upload-from-the-clipboard.ts

```diff
@@ -0,0 +1,111 @@
+import { Clipboard, getPreferenceValues, showToast, Toast } from "@raycast/api";
+import { createClient } from "webdav";
+import * as fs from "fs";
+import * as path from "path";
+
+interface Preferences {
+  webdavUrl: string;
+  username: string;
+  password: string;
+  baseDir: string;
+}
+
```

Comment:
```suggestion
```
You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`